src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1776,34 +1776,24 @@
     </Field>
     <Field Name="Designation" Type="VariableDesignationSyntax"/>
   </Node>
-  <Node Name="DeconstructionPatternSyntax" Base="PatternSyntax">
-    <Kind Name="DeconstructionPattern" />
+  <Node Name="RecursivePatternSyntax" Base="PatternSyntax">
+    <Kind Name="RecursivePattern" />
     <Field Name="Type" Type="TypeSyntax" Optional="true" />
-    <Field Name="OpenParenToken" Type="SyntaxToken">
-      <Kind Name="OpenParenToken"/>
-    </Field>
-    <Field Name="SubPatterns" Type="SeparatedSyntaxList&lt;SubpatternElementSyntax&gt;"/>
-    <Field Name="CloseParenToken" Type="SyntaxToken">
-      <Kind Name="CloseParenToken"/>
-    </Field>
+    <Field Name="DeconstructionSubpattern" Type="DeconstructionSubpatternSyntax" Optional="true" />
```

Comment:
by convention we used "Clause" to specify syntax subparts, maybe DeconstructionPatternClause, PropertyPatternClause, etc?

Reply:
I don't think Elements would be right... that's only right for tuples.

---
In reply to: [187542987](https://github.com/dotnet/roslyn/pull/26782#discussion_r187542987) [](ancestors = 187542987)

Reply:
I like "Clause". Done.

---
In reply to: [187713788](https://github.com/dotnet/roslyn/pull/26782#discussion_r187713788) [](ancestors = 187713788,187542987)